superset/tasks/schedules.py

```diff
@@ -352,7 +291,9 @@ def _get_slice_data(slc: Slice, delivery_type: EmailDeliveryType) -> ReportConte
     )
 
     cookies = {}
-    for cookie in _get_auth_cookies():
+    for cookie in WebDriverProxy(
+        driver_type=config["EMAIL_REPORTS_WEBDRIVER"]
```

Comment:
https://github.com/apache/incubator-superset/pull/10562, pushing to deprecate this one